rails/spec/requests/restoration_activity_logs_spec.rb

```diff
@@ -3,6 +3,13 @@
 
 RSpec.describe "Restoration Activity Log", type: :request do
   include_context "logged in"
+
+  describe "GET index" do
+    it "renders" do
+      get "/restoration_activity_log_entries"
```

Comment:
I realize it's probably considered redundant, but please put an expectation here, so it doesn't _look_ like a no-op test.  a simple check that the response status is `200` should suffice.